pandas/_libs/lib.pyx

```diff
@@ -134,58 +132,10 @@ def memory_usage_of_objects(ndarray[object, ndim=1] arr):
 
 #----------------------------------------------------------------------
 # datetime / io related
-
-cdef int _EPOCH_ORD = 719163
-
-from datetime import date as pydate
-
-cdef inline int64_t gmtime(object date):
-    cdef int y, m, d, h, mn, s, days
-
-    y = PyDateTime_GET_YEAR(date)
-    m = PyDateTime_GET_MONTH(date)
-    d = PyDateTime_GET_DAY(date)
-    h = PyDateTime_DATE_GET_HOUR(date)
-    mn = PyDateTime_DATE_GET_MINUTE(date)
-    s = PyDateTime_DATE_GET_SECOND(date)
-
-    days = pydate(y, m, 1).toordinal() - _EPOCH_ORD + d - 1
-    return ((<int64_t> (((days * 24 + h) * 60 + mn))) * 60 + s) * 1000
-
-
-cpdef object to_datetime(int64_t timestamp):
-    return pydatetime.utcfromtimestamp(timestamp / 1000.0)
-
-
-cpdef object to_timestamp(object dt):
-    return gmtime(dt)
-
-
-def array_to_timestamp(ndarray[object, ndim=1] arr):
```

Comment:
I don't think this is called anywhere?

Reply:
Doesn't look like it, no.  But its `def` and not `cdef`, so conceivably used downstream.  OK to remove?

Reply:
yes ok to remove

Reply:
these are ALL private modules (did that on purpose), so can move code around as long as we are internally consistent